### PR TITLE
Adding SessionStorage

### PR DESCRIPTION
**Pull request by SirDarquan**
I know this is called LocalStorage but the code lends itself to doing both. I would love to be able to handle both Local and Session storage from the same library but I'll understand if you don't want to add the functionality

### CODE REVIEW

**Comment by marcj** [comment]
Cool, very awesome contribution :) Why shouldn't I want it in the library, its very useful, thanks!

File: LocalStorage.ts [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
+
+export function LocalStorage(storageKey?: string){
+    return WebStorage(storageKey, localStorage);
+}
+
+export function SessionStorage(storageKey?: string){
+    return WebStorage(storageKey, sessionStorage);
+}
+
+function WebStorage(storageKey:string, webStorage: IWebStorage) {
```
Comment (marcj) [inline LocalStorage.ts:16]: Yes, we should rename the file then as well.

File: README.md [hunk 2]
```diff
@@ -7,13 +7,28 @@ directive (class property) using HTML5' LocalStorage.
...
+    ```
+
 
-// register LocalStorage, this registers our change-detection.
-import {LocalStorageSubscriber} from 'angular2-localstorage/LocalStorageEmitter';
-LocalStorageSubscriber(appPromise);
-```
 3. Use the `LocalStorage` decorator
 ```typescript
 import {LocalStorage} from "angular2-localstorage/LocalStorage";
```
Comment (marcj) [inline README.md]: Once renamed we should use `import {LocalStorage, SessionStorage} from "angular2-localstorage/WebStorage";`

**Comment by SirDarquan** [comment]
Great suggestion with the Readme. I totally forgot about that. Any other changes I should make?

**Comment by marcj** [comment]
Now it's perfect, thank you very much @SirDarquan! Great job.

**Pull request merged**